templates/identity/email-password/form-script-partial.html.ejs

```diff
@@ -0,0 +1,41 @@
+<script>
```

Comment:
Shouldn't this just be a `.js` then?

Reply:
That would make way more sense. I'm too clearly showcasing how often I write frontend here.